docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
+At PowerShell 7, go to 
+`C:\Users\User>` and create folder named “Neuron”: 
 
-From a powershell with **administrator** priviledges, run:
-```powershell
-PS C:\workspace\nrn> .\ci\win_download_deps.cmd
 ```
-This will download all required installers and corresponding versions(`Python`, `MPI`, `nsis`)
+mkdir Neuron
+```
 
-Then launch the installation script:
-```powershell
-PS C:\workspace\nrn> .\ci\win_install_deps.cmd
+Inside the "Neuron” folder do: 
+
+```
+git clone https://github.com/neuronsimulator/nrn
+```  
+
+ 
+Note: `PS C:\Users\User\Neuron> git clone git@github.com:neuronsimulator/nrn nrn` may be an option
+ (or may end up in a no permission issue).  
+  
+------------------------------------------------------------ 
+
+### Step 7: Downloading Dependencies
+
+As **administrator** from PowerShell 7 (right button and “Run as Administrator”) at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_download_deps.cmd
+```
+ 
+Note: You will get a number of messages of the form below; they do not indicate errors.
+
+```
+"C:\Users\User\Neuron\nrn>pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile python-3.8.exe https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe   || goto :error" 
+```
+ 
+------------------------------------------------------------ 
+
+### Step 8: Installing Dependencies
+
+As **administrator** from PowerShell 7, at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_install_deps.cmd
+``` 
+
+This command will take a while to complete.
+  
+------------------------------------------------------------ 
+
+### Step 9: Install Windows Subsystem for Linux:  wsl.exe --update  
+
+`wsl.exe --list --online` and 
+`wsl.exe --install Ubuntu-24.04` (or any other) 
+  
+------------------------------------------------------------ 
+
+### Step 10: (If you are installing neuron in a Virtual Machine) 
+
+Turn off your VM and at your computer as **administrator** put in PowerShell 7: 
+
+`
+Set-VMProcessor -VMName "NameOfYourVM" -ExposeVirtualizationExtensions $true
+`
+
+For example:
+
+```bash
+Set-VMProcessor -VMName "Windows 11 dev environment" -ExposeVirtualizationExtensions $true 
+```
+
+ 
+------------------------------------------------------------ 
+
+ 
+### Step 11: We need execute the “mingw64.exe” 
+So, look for "msys64", go to folder “msys64” and execute `mingw64.exe`.
+
+Via folder: open folder `C:\msys64` (probably) and double click in the executable `mingw64` 
+
+Via terminal: go to the location of the execuable `./mingw64.exe` (`C:\msys64` probably or `\\wsl.localhost\Ubuntu-24.04\mnt\c\msys64`) and execute it 
+
+```
+./mingw64.exe
+```
+ or `/mnt/c/msys64$ ./mingw64.exe `
+ 
+It will open a terminal. It is not PowerShell, it is not CMD, it is a mingw64 teminal.
+ 
+------------------------------------------------------------ 
+## How to build NEURON
+
+### Step 12: Creating the exe
+
+Now, inside this terminal (it is a mingw64 terminal, see step above) go to C:\Users\User\Neuron\nrn\ci and run 
```

Comment:
In most places wherer you have a path, you put it as monospaced (i.e., you used backticks). Do the same here.

Reply:
Done